pkg/pillar/cmd/tpmmgr/tpmmgr_test.go

```diff
@@ -41,6 +41,17 @@ AwEHoUQDQgAEnFLYeXVZ+SJniG0+QoqYRKDyEXx6Cs7+DyY+RcKQG16NsrkTCMXU
 -----END PRIVATE KEY-----
 `
 
+const ecdhKeyPemPkcs1 = `
```

Comment:
I thought the failure was for a pkcs1 device key and not ecdh key. (That's a nit because it is just the name for the const.)